jobs/models.py

```diff
@@ -69,7 +68,81 @@ def set_expiration_date(self):
                 self.expiration_date = self.published_date + timedelta(60)
                 self.save()
 
-
     def __unicode__(self):
         return "{0}, {1}".format(self.title, self.company)
 
+
+class Meetup(models.Model):
+
+    MEETUP = 'MEET'
+    CONFERENCE = 'CONF'
+    WORKSHOP = 'WORK'
+    MEETUP_TYPES = (
+        (MEETUP, 'meetup'),
+        (CONFERENCE, 'conference'),
+        (WORKSHOP, 'workshop'),
+    )
+
+    title = models.CharField(max_length=500)
+    organization = models.ForeignKey(
+        'Company',
```

Comment:
Maybe a better label than "company".  Jobs should have a company, but a meetup might not. Could use "Company/Organisation"?